Improvements on the documentation (#275)
mind-inria/mri-nufft: examples/trajectories/README.rst

.. _trajectories_examples:

> Trajectories Examples

Reviewer: ```suggestion
Trajectory Examples
```

---------------------

> This is a collection of examples showing features of mri-nufft to generate k-space sampling trajectories and how to display them.

Reviewer: ```suggestion
This collection of examples shows how to use MRI-nufft to generate and display k-space sampling trajectories.
```
If I recall, the preferred way is MRI-nufft right ? We should apply that everywhere (root README included)
